ojbackend/dispatcher.py

```diff
@@ -0,0 +1,78 @@
+"""
+    dispatcher.py
+    This process continuously checks whether there exist unjudged submissions.
+"""
+
+from dispatcherFunc import *
+import time
+import logging
+import subprocess, sys
+logging.basicConfig(filename = 'dispatcher.log', level = logging.INFO, format = '%(asctime)s ::%(message)s',datafmt = '%m/%d/%Y %I:%M:%S %p')
+logging.info('==========Dispatcher Started==========')
+
+machineInfo = getMachine()
+if machineInfo != None:
+    initMachine(machineInfo)
+    logging.info('getMachine Success')
+else:
+    logging.info('getMachine Error')
+    logging.info('Please check the file machineInfo.config to check the settings')
+    logging.info('==========Dispatcher Finished==========')
+    exit(0)
+
+dbIP, dbUser, dbPasswd, dbName = getdbInfo()
+if dbIP == None or dbUser == None or dbPasswd == None or dbName == None:
+    logging.info('getdbInfo Error')
+    logging.info('Please check the settings in ojdatabase.config')
+    logging.info('==========Dispatcher Finished==========')
+    exit(0)
+logging.info('connect to database')
+
+while True:
+   
+    DB = connectDB(dbIP, dbUser, dbPasswd, dbName)
+    if DB == None:
+        logging.info('connect database error')
+        logging.info('Please check the settings in ojdatabase.config')
+        logging.info('==========Dispatcher Finished==========')
+        exit(0)
+    cur = DB.cursor()
+    cur.execute("USE nthuoj;")
+    cur.execute("SELECT * FROM problem_submission WHERE status = 'WAIT' ORDER BY id ASC LIMIT 100;")
```

Comment:
Prefer to take SQL into one variable
```
subsmissionStat = "
SELECT * FROM problem_submission
WHERE status = 'WAIT'
ORDER BY id ASC
LIMIT 100;
"
```
Will be easier to maintain in the future